index.js

```diff
@@ -229,10 +237,18 @@ function register(win, accelerator, callback) {
  * @param  {BrowserWindow} win - BrowserWindow instance to unregister.
  * This argument could be omitted, in this case the function unregister the shortcut
  * on all app windows. If you registered the shortcut on a particular window instance, it will do nothing.
- * @param  {String} accelerator - the shortcut to unregister
- * @return {Undefined}
+ * @param  {String|Array<String>} accelerator - the shortcut to unregister
+ * @return {void}
  */
 function unregister(win, accelerator) {
+	if (Array.isArray(accelerator) === true) {
+		accelerator.forEach(accelerator => {
+			if (typeof accelerator === 'string') {
+				unregister(win, accelerator);
+			}
+		});
+		return;
+	}
 	let wc;
```

Comment:
`win`  argument, is optional, you should move this block after the part 
```js
if (typeof accelerator === 'undefined') {
		wc = ANY_WINDOW;
		accelerator = win;
	} else {
		if (win.isDestroyed()) {
			debug(`Early return because window is destroyed.`);
			return;
		}
		wc = win.webContents;
	}
```